nemo_skills/training/openrlhf/math_reward.py

```diff
@@ -47,7 +50,10 @@ def reward_func(queries: list[str], prompts: list[str], prompt_metadata: list[di
                 }
             )
 
-    llm = get_model(server_type="trtllm")
+    host = os.getenv("SLURM_MASTER_NODE_HET_GROUP_0", "localhost")
```

Comment:
is this assuming that the server is always running on the first het group?

Reply:
That's right. The current heterogeneous parameter in add_task is very brittle and should be used with care. A bunch of hardcoded assumptions make it quite hard to generalize, but it does work for this particular use-case